### PR TITLE
add MemoryStore

### PR DESCRIPTION
**Pull request by resigned**

### CODE REVIEW

**Comment by ianagbip1oti** [comment]
A lot of this is formatting changes.

If you run `npm run format` it should use prettify to format it consistently with the rest of the code base.

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "graphql-server-express": "1.2.0",
     "graphql-subscriptions": "0.5.4",
     "graphql-tools": "2.6.1",
+    "memorystore": "^1.6.0",
```
Comment (ianagbip1oti) [inline package.json]: I like exact versions, so with out the ^.

I've been burned by unexpected updates before :)

**Comment by resigned** [comment]
Alright, I’ll fix this and update/replace pull request

**Review by ianagbip1oti — APPROVED** [review]
👍

**Pull request merged**